ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -466,6 +473,39 @@ private ReplicaMetadataResponse getReplicaMetadataResponse(List<RemoteReplicaInf
     }
   }
 
+  /**
+   * Takes a map of store keys to store keys
+   * and creates a two way mapping between the
+   * store keys.  Assumes that input mapping
+   * maps remote keys to local keys.
+   */
+  private class TwoWayConverterCache {
```

Comment:
I find it simpler, easier, and clearer to pass around the two maps as their own class since I'm giving it to both getMissingStoreKeys and processReplicaMetadataResponse

Reply:
decided to remove the class